.travis.yml

```diff
@@ -21,6 +23,11 @@ before_install:
   - "gem update bundler"
   - "[ -f /tmp/beanstalkd-1.10/Makefile ] || (curl -L https://github.com/kr/beanstalkd/archive/v1.10.tar.gz | tar xz -C /tmp)"
   - "pushd /tmp/beanstalkd-1.10 && make && (./beanstalkd &); popd"
+  - "nvm install node"
+  - "node --version"
+  - "pushd actionview && npm install; popd"
+  - "[[ $(phantomjs --version) > '2' ]] || npm install -g phantomjs-prebuilt"
```

Comment:
We can add conditionals here on the env, so we don't waste time doing this stuff for the tests that don't care